src/templates/it/error.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base/error.html" %}
+
+{% block unknown_error %}Errore sconosciuto{% endblock %}
+
+{%
+  set localizedStatusCode = {
+    400: "Bad Request",
+    404: "Pagina non trovata",
+    500: "Errore interno del server",
+    502: "Errore temporaneo del server"
+  }
+%}
+{% macro expandedTitle(errorTitle) %}Errore: {{ errorTitle }}{% endmacro %}
+{%
+  set localizedErrorMessage = {
+    "Unsupported year requested": "Anno non supportato richiesto",
+    "Unsupported chapter requested": "Richiesto capitolo non supportato",
+    "Unsupported language requested": "Lingua richiesta non supportata",
+    "Not Found": "L'URL richiesto non è stato trovato sul server. Se hai inserito manualmente l'URL, controlla l'ortografia e riprova."
+  }
+%}
+{% macro expandedErrorMessage(errorCode, errorDescription, errorMessage) %}
+{{ errorCode }} {{ errorDescription }}: {{ errorMessage }}
```

Comment:
This will be the server description so in English:

```suggestion
{{ errorCode }} <i lang="en">{{ errorDescription }}</i>: {{ errorMessage }}
```